src/Microsoft.TestPlatform.CommunicationUtilities/EventHandlers/TestRunEventsHandler.cs

```diff
@@ -97,5 +97,12 @@ public int LaunchProcessWithDebuggerAttached(TestProcessStartInfo testProcessSta
             EqtTrace.Info("Sending LaunchProcessWithDebuggerAttached on additional test process: {0}", testProcessStartInfo?.FileName);
             return this.requestHandler.LaunchProcessWithDebuggerAttached(testProcessStartInfo);
         }
+
+        /// <inheritdoc/>
+        public bool AttachDebuggerToProcess(int pid)
+        {
+            EqtTrace.Info("Sending AttachDebuggerToProcess on additional test process with pid: {0}", pid);
```

Comment:
it is very useful when you prefix the messsage with the class and method from which it is called. in this case `TestRunEventsHandler.AttachDebuggerToProces: `.

Reply:
Done, thanks !